common/Makefile.common.mk

```diff
@@ -44,10 +44,11 @@ lint-copyright-banner:
 	@${FINDFILES} \( -name '*.go' -o -name '*.cc' -o -name '*.h' -o -name '*.proto' -o -name '*.py' -o -name '*.sh' \) \( ! \( -name '*.gen.go' -o -name '*.pb.go' -o -name '*_pb2.py' \) \) -print0 |\
 		${XARGS} common/scripts/lint_copyright_banner.sh
 
-lint-go:
-	@${FINDFILES} -name '*.go' \( ! \( -name '*.gen.go' -o -name '*.pb.go' \) \) -print0 | ${XARGS} common/scripts/lint_go.sh
+# lint-go:
```

Comment:
Let's disable individual check from here: https://github.com/IBM/ibm-common-service-operator/blob/master/common/config/.golangci.yml#L32 instead of disabling all the check

Reply:
Sure, updated